lib/dbpools.js

```diff
@@ -123,6 +123,7 @@ var configuration = {
 			if ( 1 == arrSettings[ WRITE_MASTER ] ) {
 				db_config['multipleStatements'] = true;
 				poolCluster.add( 'MISC_MASTER', db_config );
+				poolCluster.add( 'MISC_SLAVE' + slaveUniqueCount++, db_config );
```

Comment:
This is just a hack I did to overcome the parsing issues with `db-config.conf`, shouldn't be committed imo :)

Reply:
Yeah. I waffled on that. It's weird to have the `db-config.conf` file in the repo at all isn't it? Usually confs like that are a local file system thing and not committed.